mod/discussions/start.php

```diff
@@ -0,0 +1,582 @@
+<?php
+/**
+ * Discussions
+ */
+
+elgg_register_event_handler('init', 'system', 'discussion_init');
+
+/**
+ * Initialize the discussion component
+ */
+function discussion_init() {
```

Comment:
It's a bit odd that the plugin name is discussionS and the functions start with discussion